packages/components/src/styles/meta/meta.stories.ts

```diff
@@ -24,79 +21,70 @@ export default {
       url: 'https://www.figma.com/file/Um1B3fI5fvdFVJv6LO7kZG/Meta-Information?type=design&node-id=0-1&mode=design&t=I2fDQn7HjSc75K1V-0'
     }
   },
-  args: overrideArgs({ type: 'slot', name: 'default', value: '11. August 2023' }),
+  args: overrideArgs({ type: 'slot', name: 'default', value: 'Meta information' }),
   argTypes
 };
 
-/**
- * Default: This shows sd-meta in its default state.
- */
-
 export const Default = {
   render: (args: any) => {
     return generateTemplate({
-      options: { templateContent: '<time class="%CLASSES%" datetime="2023-08-11">%SLOT%</time>' },
+      options: { templateContent: '<span class="%CLASSES%">%SLOT%</span>' },
       args
     });
   }
 };
 
 /**
- * The sd-meta in all possible combinations of `variant` and `size`.
+ * Use `sd-meta` modifiers for alternative appearances.
+ *
+ * - black (default)
+ * - `sd-meta--light`: Neutral-700 can be used to deemphasize text content
  */
 
-export const LightAndSize = {
-  name: 'Light × Size',
-  parameters: { controls: { exclude: ['sd-meta--light', 'sd-meta--size-...'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'sd-meta--light' },
-        y: { type: 'attribute', name: 'sd-meta--size-...' }
-      },
-      args
-    });
-  }
+export const Variants = {
+  render: () =>
+    html`<div class="flex flex-row gap-12">
+      <span class="sd-meta">Black</span>
+      <span class="sd-meta sd-meta--light">Neutral-700</span>
+    </div>`
 };
 
 /**
- * The sd-meta in all possible combinations of `light` and `inverted`.
+ * Use `sd-meta` modifiers for alternative appearances.
+ *
+ * - large (default)
+ * - `sd-meta--size-sm`: Small can be used as an alternative in tight spaces.
  */
 
-export const LightAndInverted = {
-  name: 'Light × Inverted',
-  parameters: { controls: { exclude: ['sd-meta--light', 'sd-meta--inverted'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'sd-meta--light' },
-        y: { type: 'attribute', name: 'sd-meta--inverted', values: [false, true] }
-      },
-      args,
-      options: {
-        templateBackgrounds: { alternate: 'y', colors: ['white', 'rgb(var(--sd-color-primary, 0 53 142))'] }
-      }
-    });
-  }
+export const Size = {
+  render: () =>
+    html`<div class="flex flex-row gap-12">
+      <span class="sd-meta">Large</span>
+      <span class="sd-meta sd-meta--size-sm">Small</span>
+    </div>`
 };
 
 /**
- * The sd-meta with pipe.
+ * Use the `sd-meta--inverted` class when displayed on primary background.
+ *
+ * Once inverted meta can be used in 2 colors alternatives:
```

Comment:
I think we dont need this: "Once inverted meta can be used in 2 colors alternatives:"

Reply:
Updated